pkg/configmap/configmap.go

```diff
@@ -40,6 +41,27 @@ const (
 	syncResetBackoffMax = time.Second * 30
 )
 
+var (
+	// policyLabelKey can be replaced by --custom-label option
+	policyLabelKey   = "openpolicyagent.org/policy"
+	policyLabelValue = "rego"
+)
+
+// CustomPolicyLabel allows the default key "openpolicyagent.org/policy"
+// to be replaced by another value. This would allow two instances of kube-mgmt
+// to share a single namepace with config maps for different servers. (ie. validating & mutating)
+func CustomPolicyLabel(key, value string) (string, error) {
+	_, err := labels.NewRequirement(key, selection.Equals, []string{value})
+	if err != nil {
+		return "", err
+	}
+
+	policyLabelKey = key
```

Comment:
Hrmmm. Those values were defined as globals because they weren't changing (i.e., they ought to have been consts). I would rather make the main() function pass the values into the watcher as parameters rather than relying on mutable global state.